packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -36,6 +37,31 @@ However, if an error occurs during sign-in, Okta may need to display an error pa
 
 The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across your Okta-hosted Sign-In Widget, error pages, email templates, and Okta End-User Dashboard all at once. You don't need to set a customized Okta URL domain. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes) for more information on this feature and how to use it.
 
+## Content Security Policy (CSP) for your custom domain
+
+If you have a [custom domain](/docs/guides/custom-url-domain/main/), you need to customize the default [CSP](https://content-security-policy.com/) for your org. Okta has a default CSP that can break customizations to Okta-hosted error pages. CSP customizations only take effect on custom domains.
+
+To analyze and detect potentially malicious IP addresses that seek to bypass your CSP, use [Okta ThreatInsight](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-threatinsight).
+
+> **Note:** If you add too many trusted origins to your custom CSP, the HTTP header size can exceed the limit allowed by some server software. You may need to update the default server limits or reduce the number of trusted origins in Okta.
+
+> **Note:** If you add a trusted origin URL that redirects to a different URL, you need to include the redirect URL on the trusted origin list.
+
+### Customize the default CSP
+
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Pages** panel, click **Edit** for **Error pages**.
+3. Click **Settings**.
+4. In the **Content Security Policy** panel, click **Edit**. Set the following:
+   - **Trusted external resources**: Add resources to the CSP. For example: mydomain.com, *.mydomain.com, or https://mydomain.com/images. Click **Add**.
+     > **Note:** Okta adds these resources to all fetch-directives in the CSP.
+   - **Report UI**: Enter the URI to which you want to send violation report details. The URI entered here appears in the report-uri directive of the CSP.
+   - **Enforcement**:
+     - Select **Enforced** to block any resources that the CSP doesn't trust.
+     - Use **Not enforced** to leave the customized CSP in the report-only header. Use this option for testing and validation before turning on **Enforced** mode.
```

Comment:
maybe refer to the header as `content-security-policy-report-only` when meaning report-only header and `content-security-policy` when meaning enforce header

Reply:
done